popcorn-time/popcorn-app
# Fixed hidden popular page

How to reproduce:
- Load `popular` page
- Load any other category page
- Return to `popular` page
  Actual result:
- Page is all black
  Expected result:
- Page is loaded with all popular movies

Fix:
Use views element to find children element instead of the global accessor. This way, we make sure $page will be a children of $el. Also, use the same id for all page element, this way it won't create multiple movie-list dependant of the source (cleaner DOM, less bug prone about leftover data).
